homeassistant/components/lifx/services.yaml

```
@@ -131,12 +139,38 @@ effect_colorloop:
       domain: light
   fields:
     brightness:
-      name: Brightness
-      description: Number indicating brightness of the effect. Leave this out to maintain the current brightness of each participating light.
+      name: Brightness value
+      description: Number indicating brightness of the color loop, where 1 is the minimum brightness and 255 is the maximum brightness supported by the light.
       selector:
         number:
           min: 0
           max: 255
+    brightness_pct:
+      name: Brightness
+      description: Percentage indicating the brightness of the color loop, where 1 is the minimum brightness and 100 is the maximum brightness supported by the light.
+      selector:
+        number:
+          min: 0
+          max: 100
+          unit_of_measurement: "%"
+    saturation_min:
+      name: Minimum saturation
+      description: Percentage indicating the minimum saturation of the colors in the loop. Leave this out to use the default value of 80%.
```

```suggestion
      description: Percentage indicating the minimum saturation of the colors in the loop.
```

I think we can drop telling them what the default is since its already described below